src/components/SingleName/AddRecord.js

```diff
@@ -2,15 +2,14 @@ import React, { Component } from 'react'
 import styled from 'react-emotion'
 import { Mutation } from 'react-apollo'
 
-import { validateRecord, getPlaceholder } from '../../utils/records'
-
+import { validateRecord } from '../../utils/records'
+import DetailsItemInput from './DetailsItemInput'
 import Editable from './Editable'
 import SaveCancel from './SaveCancel'
-import DefaultInput from '../Forms/Input'
 import Select from '../Forms/Select'
 import TxPending from '../PendingTx'
 
-import { SET_CONTENT, SET_ADDRESS } from '../../graphql/mutations'
+import { SET_CONTENT, SET_OLDCONTENT, SET_ADDRESS } from '../../graphql/mutations'
```

Comment:
Instead of SET_CONTENT and SET_OLDCONTENT, we should probably change this to SET_CONTENT AND SET_CONTENTHASH